frontend/src/components/utils/Timer.jsx

```diff
@@ -19,10 +19,12 @@ export default class extends React.PureComponent {
     clearInterval(this.timer);
   }
 
+  newDate = new Date();
+
   componentDidUpdate() {
-    const time = this.props.time === undefined ? new Date() : this.props.time;
+    const time = this.props.time === undefined ? this.newDate : this.props.time;
```

Comment:
Hmm, I wonder where I get this original implementation from... it does not make any sense to me now to have the time since the first rendering... Let's refactor this in a separate PR forbidding such a use (in terms of TypeScript that would mean that we only allow `props.time` to be of `Date` type, and never undefined).